release.sh

```diff
@@ -71,7 +81,22 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+# Because we might get
+# "Error response from daemon: conflict: unable to delete 053f4edd648c (cannot be forced) - image has dependent child images"
+# we need to run in a loop to do multiple runs
+# The first one cleans up dangling images (which might be dependents), the
+# second one removes images from this build
+until docker images --quiet --no-trunc --filter "dangling=true" | \
+    ${xargs_command} docker rmi && docker images | \
+    awk '/containers.schibsted.io/ { print $3 }' | sort -u | \
+    ${xargs_command} docker rmi -f
+do
```

Comment:
Example output:

```
Untagged: containers.schibsted.io/finntech/node:onbuild-7
Untagged: containers.schibsted.io/finntech/node:onbuild-7.1
Untagged: containers.schibsted.io/finntech/node:onbuild-7.1.0
Deleted: sha256:513cd9c0c524e31f1f621540b3014af98c6ecf278fb6abedadf0f78392e961e0
Deleted: sha256:4a21b902efbb1217a4e0addee678742523e5b866adf2d6f4468cc9a437642556
Deleted: sha256:c2f096ece1b23edd820992a64219d72d01c1c72fbe7f115398c6213febeea0db
Deleted: sha256:1b8cbf261391214d962765191035e9afb12b48dc6505ac92b96a087972340160
Deleted: sha256:697a9b20407e44402575605c5fc3805be6b23e6fc85e346414a1f2136d30bdf2
Untagged: containers.schibsted.io/finntech/node:test-onbuild-7
Untagged: containers.schibsted.io/finntech/node:test-onbuild-7.1
Untagged: containers.schibsted.io/finntech/node:test-onbuild-7.1.0
Deleted: sha256:8a8edf08eeb4570767a85c1972376c2cc367effbf4d2a10221d4545149412f8d
Deleted: sha256:6357edf783285280b594ac7e1fd2738c170cc33f641560f7c156a86f76cc4b80
Deleted: sha256:49bb3f8ce190fc193d16acc5b2f4c185e03d6eac9b7fac979552d60a3f074695
Untagged: containers.schibsted.io/finntech/node:onbuild-7.7
Untagged: containers.schibsted.io/finntech/node:onbuild-7.7.0
Deleted: sha256:adbc65688c1cf0c9e5f8fe45a6ef8c364dcf3ece9909445dde00eb918daa635e
Deleted: sha256:2e56337be401835d6b7313f0c3ab8a375948a4f52a0f9d0a882fbe389cbd7d86
Deleted: sha256:644c5e1b97bf1a5bbee0f2bb823e20633066d5e8ed579d710a02fc02591213f7
Deleted: sha256:3726fac35fbbe5e868c514f6f515f48fffd3f85f18e522eb2d7a694cbf0ada20
Deleted: sha256:ca0df3615b5175fb48dab1ccb67325112fd1825fd0a149d3759f9f516bd2007b
Untagged: containers.schibsted.io/finntech/node:test-onbuild-7.7
Untagged: containers.schibsted.io/finntech/node:test-onbuild-7.7.0
Deleted: sha256:bbf1dc91e8c7fea861d581a68f3f99c9ec97ef8f0111880ef3c9d54ca41b4970
Deleted: sha256:e18b233e7fa0f992cc13a513efb0f73fb805d9c622f2e603ce98c12d39851309
Deleted: sha256:77794594d35b0cac5ee1733b1a7f28108a7c6f86a27aa7f1f5a2b94bd38c2a7b
Untagged: containers.schibsted.io/finntech/node:7.7
Untagged: containers.schibsted.io/finntech/node:7.7.0
Deleted: sha256:d36fa39f0a39cd29f7cb2a4930ac99e1a33e09cfb4d73c194c8bc3326cf9fa95
Error response from daemon: conflict: unable to delete 053f4edd648c (cannot be forced) - image has dependent child images
Error response from daemon: conflict: unable to delete 363dfed979c4 (cannot be forced) - image has dependent child images
Error response from daemon: conflict: unable to delete 5711735b3d42 (cannot be forced) - image has dependent child images
Untagged: containers.schibsted.io/finntech/node:7
Untagged: containers.schibsted.io/finntech/node:7.1
Untagged: containers.schibsted.io/finntech/node:7.1.0
Deleted: sha256:053f4edd648c4224e98e3cfdd142cdc518fa779e87421ed3b316bdfdc96f50bc
Untagged: containers.schibsted.io/finntech/node:test-7.7
Untagged: containers.schibsted.io/finntech/node:test-7.7.0
Deleted: sha256:363dfed979c474224408076b4cbdab33c0ecd09d90f4b11b72409de63240a4a5
Deleted: sha256:03f36ba8dd2c014a2cfef47b5b8d93906b9b59eae05fcecefc557fcea893efbf
Deleted: sha256:01e6bd05040f8e0628c946563c05611f04452cd480e66fde250f03c22b8f6094
Untagged: containers.schibsted.io/finntech/node:test-7
Untagged: containers.schibsted.io/finntech/node:test-7.1
Untagged: containers.schibsted.io/finntech/node:test-7.1.0
Deleted: sha256:5711735b3d4291e472daf24b5c593daff85e7529f2f102884ed4bdf78da6bb13
Deleted: sha256:0820ea90c14ba59446f5fd45e166fe5e6aadb942bdab012954279d3902bca5d4
Deleted: sha256:76057c0d2af545fe21409442766949c9f95c5bceaed634b4bea831de6cbfdd21
```

The tricky part is the `Error response from daemon: conflict: unable to delete 5711735b3d42 (cannot be forced) - image has dependent child images` and friends, I still need the loop (I think)